app/views/directives/resource-service-bindings.html

```diff
@@ -0,0 +1,28 @@
+<div class="expanded-section mar-bottom-xl" ng-if="$ctrl.showBindings">
+  <h3>Service Bindings</h3>
+  <overview-service-binding
+    ng-if="($ctrl.bindings | size)"
+    ng-repeat="binding in $ctrl.bindings track by (binding | uid)"
+    binding="binding"
+    service-classes="$ctrl.serviceClasses"
+    service-instances="$ctrl.serviceInstances"
+    secrets="$ctrl.secrets">
+  </overview-service-binding>
+  <div ng-if="($ctrl.bindableServiceInstances | size) && ({resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create')">
+    <a href="" ng-click="$ctrl.createBinding()" role="button">
+      <span class="pficon pficon-add-circle-o"></span>
+      Create Binding
+    </a>
+  </div>
+  <div ng-if="!($ctrl.bindableServiceInstances | size)">
+    <span>You must have a bindable service in your namespace in order to create bindings.</span>
+    <div>
+      <a href="/">Browse Catalog</a>
```

Comment:
I believe this should be

```
<a href="./">Browse Catalog</a>
```

at least according to

https://github.com/openshift/origin-web-console#navigation